.github/workflows/release.yml

```
@@ -0,0 +1,51 @@
+          fetch-depth: 0
+      - uses: ./.github/actions/common-setup
+
+      - name: test
+        run: npm run test
+
+      - name: publish
+        run: npm publish # Publishes to npmjs, using package.json's version
+
+      - name: version
+        id: version
+        run: |
```

actionlint (1.7.10)

shellcheck reported issue in this script: SC2086:info:4:22: Double quote to prevent globbing and word splitting

(shellcheck)